traffic_portal/app/src/common/api/ServiceCategoryService.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var ServiceCategoryService = function($http, ENV, locationUtils, messageModel) {
+
+    this.getServiceCategories = function(queryParams) {
+        return $http.get(ENV.api['root'] + 'servicecategories', {params: queryParams}).then(
```

Comment:
please change these to be `ENV.api['latest']`.  once PR 4633 gets merged that will be API v3 and root will still be v2.  You can also hardcode in /api/3.0 if you want with a TODO to fix it later if PR4633 isnt merged as soon as we would like this to be